instances/anatomicalPlane/sagittal.jsonld

```diff
@@ -4,7 +4,7 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/anatomicalPlane/sagittal",
   "@type": "https://openminds.ebrains.eu/controlledTerms/AnatomicalPlane",
-  "definition": "An anatomical plane which divides the body into right and left parts.",
+  "definition": "An anatomical plane which divides the body or parts of the body into left and right (and/or medial and lateral) parts.",
```

Comment:
Suggestion: The 'sagittal plane' refers to any plane that runs parallel to the 'median plane' which divides a symmetrical organism or organ into it's left and right parts.

Reply:
Regarding the 'sagittal plane', be aware of the 'neurological versus radiological' display convention (see link: https://nipy.org/nibabel/neuro_radio_conventions.html). 
In short, "Neurologists like looking at brain images with the patient’s right on the right of the image" versus "Radiologists like looking at their images with the patient’s left on the right of the image".

At the moment there are no issue as you've stated "...organism or organ into it's left and right parts", and thus following the neurological display convention, however it is useful to be aware.

Reply:
Too keep up with the two other axes, should this be renamed to Longitudinal?

Suggestion:
The 'longitudinal plane' is always perpendicular to the axial and frontal axes of an organism, extremity or organ, running parallel to the 'median plane' and dividing it into a left and right part. 

Link to webpage: https://www.healthline.com/health/body-planes#sagittal-plane
We could use this as a reference too, but I don't know: https://www.frontiersin.org/articles/10.3389/fninf.2023.1154080/full

Could we add synonym?

"synonym": [
"Sagittal"
]